trainingsanleitungen/scratch/scratch-space-shooter-v3.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: sushi
+title: Scratch Space Shooter
+description: In dieser Übung schießt du mit deinem Raumschiff herabfallende Meteoriten ab bevor sie dein Raumschiff zerstören.
+---
+
+# Scratch Space Shooter
+
+In dieser Übung schießt du mit deinem Raumschiff herabfallende Meteoriten ab bevor sie dein Raumschiff zerstören.
+
+## Bühne und Figuren anlegen
+
+1. ![Bühnenbild und Figuren anlegen](scratch-space-shooter-v3/01-background.png){: .right}
+Als erstes legst du fest, wie deine Bühne aussehen soll. Für dieses Spiel brauchst du das Weltall als Hintergrund. Wähle ein passendes Bild aus oder male selbst eines.
+
+2. ![Raumschiff](scratch-space-shooter-v3/02-spaceship.png){: .right}
+Als erste Figur brauchst du das Raumschiff. Es besteht aus zwei Kostümen: dem Raumschiff selbst und einem weiteren Kostüm das angezeigt wird, wenn das Spiel vorbei ist. Zeige den Text "Game Over" an oder male ein passendes Bild
+
+3. ![Laser](scratch-space-shooter-v3/03-laser.png){: .right}
+Die nächste Figur ist der Laserstrahl, der von der Rakete abgefeuert werden kann. Diesen kannst du dir selbst malen.
+
+4. ![Meteroit](scratch-space-shooter-v3/04-meteorit.png){: .right}
+Und als letzte Figur brauchst du noch einen Meteoriten. Diesen kannst du dir selbst malen.
+
+## Daten
+
+1. ![Daten](scratch-space-shooter-v3/05-data.png){: .right}
+Wir brauchen für den Space Shooter einige Daten: die Anzahl der Treffer, die Enstehungszeit von Meteoriten, deren Geschwindigkeit und einen Indikator, ob ein Meteroit getroffen wurde. ACHTUNG: die Variable getroffen gilt nur für die Figur "Meteroit", alle anderen Variablen gelten für alle Figuren.
```

Comment:
Hier wäre es schön, statt *ACHTUNG* das normal geschriebene Wort *Achtung* zu verwenden, allerdings mit zwei Sternchen rundherum: `**Achtung**`. Das führt dazu, dass *Achtung* in fetter Schrift angezeigt wird.